src/aiomealie/models.py

```diff
@@ -142,6 +144,34 @@ class ShoppingItem(DataClassORJSONMixin):
     unit_id: str = field(metadata=field_options(alias="unitId"))
 
 
+@dataclass
+class MutateShoppingItem(DataClassDictMixin):
+    """MutateShoppingItem model."""
+
+    item_id: str | None = field(default=None, metadata=field_options(alias="id"))
+    list_id: str | None = field(
+        default=None, metadata=field_options(alias="shoppingListId")
+    )
+    note: str | None = field(default=None)
+    display: str | None = field(default=None)
+    checked: bool | None = field(default=None)
+    position: int | None = field(default=None)
```

Comment:
```suggestion
    note: str | None = None
    display: str | None = None
    checked: bool | None = None
    position: int | None = None
```
I know this works to set it as None, but not sure if this works the same with `code_generation_options = ["TO_DICT_ADD_OMIT_NONE_FLAG"]`